Features/ModerationCommands/BanCommand.cs

```diff
@@ -75,8 +68,26 @@ await _databaseService.NewBanRecordAsync(
                 Timestamp = DateTime.UtcNow
             };
 
+            var dmEmbed = new DiscordEmbedBuilder
+            {
+                Title = "You Have Been Banned",
+                Color = DiscordColor.Red,
+                Description = $"**Server:** {ctx.Guild.Name}\n" +
+                            $"**Reason:** {reason}\n" +
+                            $"**Appeals:** If you believe this ban was a mistake, please reach out to the moderation team for clarification.",
+                Thumbnail = new DiscordEmbedBuilder.EmbedThumbnail
+                {
+                    Url = targetUser.AvatarUrl
```

Comment:
I think it's smarter to show the server icon instead of the user icon, if anything at all